version/current_test.go

```diff
@@ -28,27 +27,15 @@ func (*CurrentSuite) TestCurrentSeries(c *gc.C) {
 	if err != nil {
 		// If the command fails (for instance if we're running on some other
 		// platform) then CurrentSeries should be unknown.
-		switch runtime.GOOS {
-		case "darwin":
-			c.Check(s, gc.Matches, `mavericks|mountainlion|lion|snowleopard`)
-		default:
-			currentOS, err := corebase.GetOSFromSeries(s)
-			c.Assert(err, gc.IsNil)
-			if s != "n/a" {
-				// There is no lsb_release command on CentOS.
-				if currentOS == os.CentOS {
-					c.Check(s, gc.Matches, `centos\d+`)
-				}
-			}
-		}
-	} else {
-		//OpenSUSE lsb-release returns n/a
 		currentOS, err := corebase.GetOSFromSeries(s)
 		c.Assert(err, gc.IsNil)
-		if string(out) == "n/a" && currentOS == os.OpenSUSE {
-			c.Check(s, gc.Matches, "opensuseleap")
-		} else {
-			c.Assert(string(out), gc.Equals, "Codename:\t"+s+"\n")
+		if s != "n/a" {
```

Comment:
Does it make sense to just combine all these if statements together now?

Reply:
Done